index.js

```diff
@@ -5,20 +5,209 @@ const MobileNavImg = document.querySelector('.mobile-nav-img');
 const IconClose = document.querySelector('.icon-close');
 
 const NavbarOptions = document.querySelectorAll('.navbar-options');
-
 const HeadlineWrapper = document.querySelector('.headline');
 const logo = document.querySelector('.logo');
+const projects = document.querySelector('.work-collection');
+const about = document.querySelector('.about-myself');
+const contact = document.querySelector('.contact-form');
+
+// const SnapshotImage = document.querySelector('#snapshot');
+// const ProjectName = document.querySelector('#projectname');
+
+fetch('projects.json')
+  .then((response) => response.json())
+  .then((data) => {
+    data.forEach((project, index) => {
```

Comment:
- [ ] Please avoid the use of the `fetch` API at this point you even don't need it, you can use the module feature of JavaScript and easily import the array of object from the `project.json` file and use it instead of making a `HTTP` request and wait for the result. Because `fetch` API have very less support in old browser and you are importing file internally so it is not a good way for getting the data and display it.